### DISCUSSION TITLE
mark  question

### DISCUSSION THREAD

**Discussion by ghost** [pytest-dev/pytest#8479] [Q&A] [answered]
I have two test cases here with the same name, but the mark of each use case is different. After using the mark to execute the use case, only the second use case is always executed. May I ask: Must the use case name not be repeated?

**Comment by mrbean-bremen** [comment, marked as answer]
If you use the same test name twice in the same scope, the second simply replaces the first one (as would be the case with any Python function, e.g. the new function is assigned to the name). This is independent of marks.
